modules/ROOT/pages/persistence-gateway.adoc

```diff
@@ -90,6 +90,18 @@ The output of this command should be similar to the following:
 192.168.2.101 - - [09/Apr/2021:16:35:55 +0000] "GET /api/v1/status/ready HTTP/1.1" 200 2 "" "kube-probe/1.18+"
 ----
 
+== Configure Authorize Namespace
+
+Edit the RoleBiding that you create during the configuration of xref:install-helm.adoc#optional-configure-authorized-namespaces[authorized namespaces] by adding a `subjects` object:
```

Comment:
```suggestion
Edit the RoleBinding that you create during the configuration of xref:install-helm.adoc#optional-configure-authorized-namespaces[authorized namespaces] by adding a `subjects` object:
```

Reply:
They don't need to add the subjects field, it's already there. They just need to add a new subject.